lib/ansible/modules/cloud/ovirt/ovirt_hosts.py

```diff
@@ -336,6 +341,7 @@ def main():
         override_display=dict(default=None),
         kernel_params=dict(default=None, type='list'),
         hosted_engine=dict(default=None, choices=['deploy', 'undeploy']),
+        iscsi=dict(default=None, type='dict'),
```

Comment:
The request should be as follow:
- ovirt_hosts:
    state: iscsidiscover
    name: myhost
    iscsi:
         username: iscsi_user
         password: *****
         address: 10.34.61.145
How do you want to rename it?